test/e2e/view-internal-server.js

```diff
@@ -1,22 +0,0 @@
-const expect = require('chai').expect
```

Comment:
Are we unable to cause a 500 error by any other means? I would usually take that as a good sign!
I think we added this test because it was so easy to create a 500 this way

Reply:
I couldn't think of another known scenario for a 500 unfortunately